enclave-runtime/Cargo.lock

```diff
@@ -1886,7 +1886,7 @@ dependencies = [
  "parity-scale-codec",
  "serde 1.0.118 (git+https://github.com/mesalock-linux/serde-sgx?tag=sgx_1.1.3)",
  "serde_json 1.0.60 (git+https://github.com/mesalock-linux/serde-json-sgx?tag=sgx_1.1.3)",
- "sgx_crypto_helper",
+ "sgx_crypto_helper 1.1.6 (git+https://github.com/apache/incubator-teaclave-sgx-sdk?branch=master)",
```

Comment:
When you suddenly have additional information in the dep here, it means that we have the dependency twice. So we are probably still referencing this crate from the `v1.1.6-testing` branch somewhere.

Reply:
Should be resolved now